pages/dkp/kommander/2.2/dkp-upgrade/index.md

```diff
@@ -1,10 +1,62 @@
 ---
 layout: layout.pug
-navigationTitle: DKP upgrade
-title: How to upgrade DKP
+navigationTitle: Upgrade DKP
+title: How to upgrade DKP via CLI
 menuWeight: 8
-excerpt: Upgrade DKP to the latest version and benefit from the full DKP experience.
+excerpt: Upgrade DKP to the latest version
 beta: false
 ---
 
-This section describes how to upgrade your environment to the latest version of DKP.
+The DKP upgrade represents an important step of your environment's lifecycle, as it ensures that you are up-to-date with the latest features and can benefit from the latest improvements, enhanced cluster management, and better performance. This section describes how to upgrade your networked, air-gapped or on-prem environment to the latest version of DKP.
+
+## Understand the upgrade process
+
+For this release, you perform the upgrade sequentially beginning with DKP Kommander and then moving to DKP Konvoy.
+
+When upgrading DKP, the process is different depending on if you run a stand-alone Management Cluster, or if you run a multi-cluster environment that includes a combination of a Management Cluster and Workspace Clusters (Attached or Managed Clusters). 
+
+Start with your Management Cluster in Kommander, and then proceed, Workspace by Workspace, if any. Then you can move to upgrading Konvoy, cluster by cluster.
+
+The overall process for upgrading to the latest version of DKP is done on each Workspace or cluster, and has the following high-level steps;
+
+   For **Kommander**, on your Management Cluster:
+
+1.  [**Upgrade Kommander**][upgrade_kommander], which upgrades all Platform Applications.
+
+    If you do not have any Managed or Attached Clusters, skip to upgrading Konvoy on your Management Cluster.
+    
+
+   For **Kommander**, on your Workspaces (which include Management and Managed clusters):
+
+1.  [**Upgrade Kommander on your Workspaces**][upgrade_workspaces], which upgrades all Platform Applications on your Workspace Clusters (Workspace Clusters can include Attached and Managed Clusters).
+
+1.  [**Upgrade all DKP Catalog applications**][catalog_apps] deployed to any Workspaces or Projects.
+
+1.  [**Verify any Custom Catalog applications**][custom_apps] and ensure they are compatible with the current Kubernetes version.
```

Comment:
'current Kubernetes version' is ambiguous.   I think we should point at the release notes and say something like:

```suggestion
1.  [**Verify any Custom Catalog applications**][custom_apps] and ensure they are compatible with the Kubernetes version included in the new release.
```